<launch>

  <!-- Urg Node -->
  <node pkg="urg_node" name="urg_node" type="urg_node">
    <param name="ip_address" value="192.168.0.10"/>
    <param name="angle_min" value="-1.832595714595"/>
    <param name="angle_max" value=" 1.832595714595"/>
  </node>

  <node pkg="tms_ss_pot" type="change_nan" name="change_nan">
    <remap from="/LaserTracker" to="/LaserTracker1"/>
  </node>

  <!-- Tracker and Visualizer -->
  <node pkg="tms_ss_pot" name="urg_single_tracker" type="urg_single_tracker" output="screen" />
  <node pkg="tms_ss_pot" name="visualize" type="visualize" />

  <!-- TF -->
  <node pkg="tf" type="static_transform_publisher" name="world_2_map" args="0 0 0
  0 0 0 /world_link /map 100"/>

  <!-- Map server -->
  <arg name="map_file" default="$(find tms_ss_pot)/maps/map_bsen_p/map.yaml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)">
  <param name="frame_id" value="map" />
  </node>

  <!-- RViz -->
  <node name="rviz_tracker" pkg="rviz" type="rviz" args="-d $(find tms_ss_pot)/rviz/tracker_config.rviz" respawn="true" />

</launch>
